osf_tests/metadata/expected_metadata_files/agent_supplementary.turtle

```diff
@@ -0,0 +1 @@
+
```

Comment:
Should anything be in here, or is this more of a placeholder for if/when User Profiles get a rewrite and have e.g. view counts?

Reply:
makes sense to call it a placeholder, yeah -- as implemented, empty supplement is valid and i figure the machinery should work for users even tho there are no particular plans for supplementary metadata on users

...there could in theory be monthly view counts for the user profile page, just checked it does already POST a counted-usage, buuuut (for [some reason](https://github.com/CenterForOpenScience/osf.io/blob/decc312a2be6111f1faccfd708d137a5d69343a7/website/static/js/keen.js#L121-L126)) always sets `item_public: false`, so the monthly reporter will always skip it